### ISSUE TITLE
Design: Yeast Collection

### ISSUE THREAD

**Issue by vinoo-igem** [iGEM-Engineering/iGEM-distribution#28]
This will be a larger project that will need to get subdivided into smaller tasks/sub-collections. It would be great to start with what are considered as absolute essentials.

**Comment by noahsprent** [comment]
Not sure if it's an absolute essential, as I imagine by yeast you probably mean _S. cerevisiae_, but for _Pichia pastoris_ this should be a good source: https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-017-0492-3. Might be able to contribute to adding this to a package/subpackage later if needs be but figured I'd throw it in here for discussion of whether or not to include it first.

**Comment by noahsprent** [comment]
Kit on addgene: https://www.addgene.org/kits/gasser-goldenpics/#kit-contents

**Comment by noahsprent** [comment]
Seems like it would be worth including the Open Yeast Collection, which is already on freegenes, here: https://stanford.freegenes.org/products/open-yeast-toolkit#genes. See also #31

**Comment by noahsprent** [comment]
I have made a branch, open-yeast-collection, and added the Open Yeast Collection to the spreadsheet. Waiting on #77 and #31 for the spreadsheet to pass tests though, I think, as I added a FreeGenes row to the data sources sheet. So we'll need to add code to grab genbank files from a generic URI as well as the code to understand what the URI is for a specific FreeGenes ID.

**Comment by jakebeal** [comment]
Actually, it looks like the checks are failing right now because SynBioHub was down. I kicked them to restart, and they are passing the checks now, so I think that you can still go ahead and set up a pull request. The package won't be complete until the issues are resolved, but it can move forward to review by others of its parts content, at least.

**Comment by noahsprent** [comment]
Some notes from Scott Pownall, lead creator of the OYC, on it's inclusion:

> Note that the FreeGenes assembly standard that is used in OYC is not yeast specific and many of the parts are agnostic to species (E.coli backbone, assembly connectors). This standard was designed by myself and Keoni with feedback from Isaac and others. Overhangs were chosen for optimal assembly. The parts are sanitized for the following restriction enzymes except they are needed - AarI, BamHI, BbsI, BglII, BsaI, BsmBI, BtgZI, EcoRI, NheI, NotI, NruI, PstI, SapI, SpeI, XbaI, XhoI, HindIII.

**Comment by noahsprent** [comment]
Another point is that he has designed two fluorescent reporters that are codon optimized for yeast - one is mCherry, whose patent has expired, and the other is the open fuGFP. He was going to self-fund the synthesis but we could potentially include these in the distribution? If we don't have to synthesise the OYC then these would be the only yeast parts being synthesised. I don't suppose they've been characterised though if they haven't been synthesised yet.

**Comment by jakebeal** [comment]
I've opened an update in #107 that imports all of the sequences from FreeGenes

**Comment by noahsprent** [comment]
If we want to add more yeast parts we should open a new issue that's more specific about what we want.
